macros/cross_db_utils/escape_single_quotes.sql

```diff
@@ -0,0 +1,15 @@
+{% macro escape_single_quotes(expression) %}
+      {{ return(adapter.dispatch('escape_single_quotes', 'dbt_utils') (expression)) }}
+{% endmacro %}
+
+{% macro default__escape_single_quotes(expression) -%}
+{{ expression | replace("'","''") }}
```

Comment:
not blocking but we might want to explain these arcane escaping rules just a tad